### PR TITLE
Update CLI output to match documentation

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum <tim.mccallum@fermyon.com>

Update console output to match the Quickstart documentation (adjust paths to resources)

### CODE REVIEW

File: charts/spin-operator/templates/NOTES.txt [hunk 1]
```diff
@@ -13,11 +13,11 @@ already done so, please ensure the following:
 
 1. Install the containerd-shim-spin SpinAppExecutor:
 
-  $ kubectl apply -f https://raw.githubusercontent.com/spinkube/spin-operator/main/config/samples/spin-shim-executor.yaml
+  $ kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.1.0/spin-operator.shim-executor.yaml
```
Comment (rajatjindal) [inline charts/spin-operator/templates/NOTES.txt:16]: we should be able to use a variable here instead of hardcoded release number.

**Comment by rajatjindal** [comment]
requesting review from @adamreese / @vdice

File: charts/spin-operator/templates/NOTES.txt (hunk 1 above)
Comment (tpmccallum) [reply charts/spin-operator/templates/NOTES.txt]: Great point @rajatjindal 

I see that the chart version does not include the `v` and therefore the update leaves the `v` in place and adds the chart version after it.

Screenshot 2024-04-11 at 13 04 12


Screenshot 2024-04-11 at 13 02 37

**Review by vdice — APPROVED** [review]

**Review by adamreese — APPROVED** [review]

**Review by michelleN — APPROVED** [review]

**Review by michelleN — COMMENTED** [review]
Thanks @tpmccallum. This will help users get a compatible version of the executor and runtime class!

**Pull request merged**